frontend/src/routes/(root)/(logged)/resources/+page.svelte

```diff
@@ -75,7 +75,8 @@
 	let newResourceType = {
 		name: '',
 		schema: emptySchema(),
-		description: ''
+		description: '',
+		isValid: false
```

Comment:
this should be held separately since it's not a property of the resource type that we want to send to the api